scripts/coverage.sh

```diff
@@ -1,136 +1,69 @@
 #!/usr/bin/env bash
 #
-# Runs all tests and collects code coverage
+# Run tests and collect code coverage
 #
-# Warning: this process is a little slow
+# Run all:
+#   $ ./script/coverage.sh
 #
+# Run for specific packages
+#   $ ./script/coverage.sh -p solana-account-decoder
+#   $ ./script/coverage.sh -p solana-account-decoder -p solana-accounts-db [-p ...]
 
-if ! command -v grcov; then
-  echo "Error: grcov not found.  Try |cargo install grcov|"
-  exit 1
-fi
-
-if [[ ! "$(grcov --version)" =~ 0.8.[0-9] ]]; then
-  echo Error: Required grcov version not installed
+set -e
+here=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
 
-  echo "Installed version: $(grcov --version)"
+# Check for grcov commands
+if ! command -v grcov >/dev/null 2>&1; then
+  echo "Error: grcov not found.  Try |cargo install grcov|"
   exit 1
 fi
 
-set -e
-cd "$(dirname "$0")/.."
-source ci/_
+# Use nightly as we have some nightly-only tests (frozen-abi)
+# shellcheck source=ci/rust-version.sh
+source "$here/../ci/rust-version.sh" nightly
 
-cargo="$(readlink -f "./cargo")"
+# Clean up
+rm -rf "$here/../target" "$here/../coverage"
```

Comment:
hmm, i actually liked the old behavior of using `target/cov`. Could you restore it?

I think this implicit use of `target/debug` invalidates all other build caches. Also, I `rm -rf ./target/` is a bit intrusive for local development. At the very least, it could be restricted to `./target/cov`? Better yet, I don't want to trash build cache everytime. is this really needed? I think the following 2 `find`s was enough?

Reply:
should be fixed in [57f67c0](https://github.com/anza-xyz/agave/pull/1260/commits/57f67c082e0fc05be4723ca9deb5392e4dc81ce2)